nni/retiarii/nn/pytorch/api.py

```diff
@@ -64,9 +65,18 @@ def create_fixed_module(cls, candidates: Union[Dict[str, nn.Module], List[nn.Mod
                             label: Optional[str] = None, **kwargs):
         chosen = get_fixed_value(label)
         if isinstance(candidates, list):
-            return candidates[int(chosen)]
+            result = candidates[int(chosen)]
         else:
-            return candidates[chosen]
+            result = candidates[chosen]
+
+        # map the named hierarchies to support weight inheritance for python engine
```

Comment:
have not supported value choice, right?

Reply:
Value choice are "values", there is no weight to load into value choice.